deployment/scripts/.env

```diff
@@ -11,8 +11,11 @@ pvc_storage_requests="3Gi"
 
 # Ingress Environment variables
 letsencrypt_email="helloworld@gmail.com"
-ingress_dns="example.com"
-ingress_http_port="80"
+fuel_core_ingress_dns="example.com"
+fuel_core_ingress_http_port="80"
+
+# Monitoring Environment variables
+grafana_ingress_dns="helloworld@monitoring.com"
```

Comment:
should this be a regular domain like `example.com` instead of an email?

Reply:
@Voxelot - you are right- updated just now